OutOfSchool/OutOfSchool.RazorTemplatesData/Config/EmailContentConfig.cs

```diff
@@ -0,0 +1,20 @@
+namespace OutOfSchool.RazorTemplatesData.Config;
+
+public class EmailContentConfig
+{
+    private readonly string EmailContentHost;
+    private static readonly string EmailContentPath = "/_content/email";
+
+    public string EmailContentUrl
+    {
+        get
+        {
+            return string.Concat(EmailContentHost, EmailContentPath);
+        }
+    }
```

Comment:
(Code style) Please use less verbose syntax for simple getter-only properties. I.e.
```
public string EmailContentUrl => string.Concat(EmailContentHost, EmailContentPath);
```